packages/shared/src/hooks/useCopyLink.ts

```diff
@@ -9,6 +9,7 @@ type CopyNotifyFunctionProps = NotifyOptionalProps & {
 };
 
 const defaultMessage = '✅ Copied link to clipboard';
+const noLinkErrorMessage = '❌ Could not copy, link is missing';
```

Comment:
There is a bunch of string concatenations around the project for the links eg. `${post?.commentsPermalink}${getCommentHash(commentId)}` where if something is `undefined` the link would end up `undefined/bCrZekKfq` or `undefined/undefined`. 

I think the correct way to handle this is to return `undefined` from the `getLink` method like I did in this PR. And then this is why I added additional handling so it can be caught in development and production.